src/mpid/ch3/include/mpid_rma_issue.h

```diff
@@ -801,14 +712,17 @@ static int issue_get_acc_op(MPIDI_RMA_Op_t * rma_op, MPIR_Win * win_ptr,
         rest_len -= stream_size;
 
         /* Set extended packet header if needed. */
-        init_get_accum_ext_pkt(flags, target_dtp_ptr, stream_offset, &ext_hdr_ptr, &ext_hdr_sz);
+        init_accum_ext_pkt(flags, target_dtp_ptr, stream_offset, &ext_hdr_ptr, &ext_hdr_sz,
+                           &get_accum_pkt->info.flattened_type_size);
 
         /* Note: here we need to allocate an extended packet header in response request,
          * in order to store the stream_offset locally and use it in PktHandler_Get_AccumResp.
          * This extended packet header only contains stream_offset and does not contain any
          * other information. */
-        init_get_accum_ext_pkt(flags, NULL /* target_dtp_ptr */ , stream_offset,
-                               &(resp_req->dev.ext_hdr_ptr), &(resp_req->dev.ext_hdr_sz));
+        int dummy;
+        init_accum_ext_pkt(flags, NULL /* target_dtp_ptr */ , stream_offset,
+                           &(resp_req->dev.ext_hdr_ptr), &(resp_req->dev.ext_hdr_sz),
+                           &dummy);
 
```

Comment:
The `dummy` declaration should be moved to the beginning of the code block.

Reply:
We decided to allow C99 now, isn't it?

Reply:
I am not sure...I do not see the style in other files. @raffenet ?

Reply:
I'm pretty sure we decided to allow C99 everywhere now, not just in `ch4`.

Reply:
We should probably go and update this link based on what @raffenet says: https://wiki.mpich.org/mpich/index.php/Coding_Standards#Adopting_C99_and_newer_POSIX_standards

Reply:
C99 everywhere, yes.

Reply:
Thanks!  I've updated the wiki page.